buildSrc/src/main/java/Parser.java

```diff
@@ -168,9 +168,6 @@ private char fixup(int intChar) {
         switch (intChar) {
             case 8217: // back-tick `
                 return '\'';
-            case 8221: // fancy open-quotes ”
```

Comment:
case 8221: // fancy open-quotes ”
            case 8220: // fancy close-quotes “
                return '\"';

Reply:
Sorry for this, we thought that we had added those lines unintentionally, so we deleted. And now, we realized that there was a character that it was wrong.